src/main/java/codes/ai/process/LocalServerProcess.java

```diff
@@ -0,0 +1,151 @@
+package codes.ai.process;
+
+import com.google.common.collect.ImmutableList;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author xuy.
+ *         Copyright (c) Ai.codes
+ */
+public class LocalServerProcess {
+  private static List<String> nodePaths = ImmutableList.of(
+      "/usr/local/bin/node", "/opt/local/bin/node", "/usr/bin/node", "C:\\Program Files\\Nodejs", "C:\\Program Files (x86)\\Nodejs");
+  private static String scriptName = "server.js";
+  private final String basedir;
+  private Process process;
+  private Thread outThread;
+  private Thread errThread;
+  
+  public LocalServerProcess(String pluginBaseDir) {
+    this.basedir = pluginBaseDir;
+  }
+  
+  public void start() {
```

Comment:
Yes, the plugin has its own "Activity" class (still investigating) that has a start() hook. Today I will try to get the local_proxy code to a state that can be invoked, and then call this start() in IntelliJ's plugin activity start(), so the plugin gets the node local proxy running at startup time. Error detection, auto restart after crash etc will come later.